chat-components/src/components/common/commandbutton/CommandButton.tsx

```diff
@@ -66,6 +66,7 @@ function CommandButton(props: ICommandButtonProps) {
                     disabled={props.disabled}
                     className={props.className}
                     styles={buttonStyles}
+                    title={props.hideButtonTitle ? undefined : props.ariaLabel}
```

Comment:
Why are we using title not ariaLabel?

Reply:
I can  add the aria label, but for what I see , the aria label is generally set at specific component level (Email, download, etc)